### PR TITLE
fix dependencies version lock.

### PR DESCRIPTION
**Pull request by laixintao**
`pytest~=6.2` means `6.*` instead of `6.2.*`

### CODE REVIEW

**Comment by laixintao** [comment]
it looks good now, but the only problem is...

every log in CI was printed 3 times... don't know why

image

**Comment by laixintao** [comment]
> it looks good now, but the only problem is...
> 
> every log in CI was printed 3 times... don't know why
> 
> image

It looks like a github action issue, you will see 3 duplicated output when the action is running, but after its done, the output is ok.

File: pyproject.toml [hunk 1]
```diff
@@ -34,17 +34,17 @@ repository = "https://github.com/laike9m/pdir2"
...
-    "tox-gh-actions~=2.8",
+    "pytest==6.2.*",
+    "tox==3.24.*",
+    "tox-pdm==0.3.*",
+    "ptpython==3.0.*",
+    "bpython==0.21.*",
+    "ipython==7.16.*",
+    "flake8==3.9.*",
+    "pytest-mypy==0.8.*",
+    "pandas==1.3.5",
```
Comment (laixintao) [inline pyproject.toml:45]: some of the versions are not working anymore, so I have to manually fix them by locking the version to a working one. it's painful TT

**Comment by laike9m** [comment]
Thanks a lot!

**Pull request merged**